open-metadata-resources/open-metadata-deployment/docker/ranger/Dockerfile

```diff
@@ -66,109 +30,77 @@ RUN mvn -Pall -DskipTests=true clean compile package install assembly:assembly
+RUN chmod 755 /opt/ranger-admin/ranger-docker-start.sh
+
+# Configure and install Solr locally
+WORKDIR /opt/ranger-admin/contrib/solr_for_audit_setup
 RUN sed -i 's|SOLR_INSTALL=false|SOLR_INSTALL=true|' install.properties
-RUN sed -i "s|SOLR_DOWNLOAD_URL=|SOLR_DOWNLOAD_URL=http://www.mirrorservice.org/sites/ftp.apache.org/lucene/solr/${solrver}/solr-${solrver}.tgz|" install.properties
-
-#Update path to include java, so that when we run the container it is available
-#default shell isn't bash :-(
-
-#RUN echo "export JAVA_HOME=/usr/lib/jvm/default-java" >> /root/.bashrc
-RUN echo "export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64" >> /root/.profile
-RUN echo "export PATH=${JAVA_HOME}/bin:$PATH" >> /root/.profile
-RUN chmod +x /root/.profile
+RUN sed -i "s|SOLR_DOWNLOAD_URL=|SOLR_DOWNLOAD_URL=http://www.mirrorservice.org/sites/ftp.apache.org/lucene/solr/7.7.1/solr-7.7.1.tgz|" install.properties
+RUN ./setup.sh
+RUN sed -i 's|^SOLR_JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"$|SOLR_JAVA_HOME="/usr/lib/jvm/java-1.8-openjdk"|g' /opt/solr/ranger_audit_server/scripts/solr.in.sh
 
-
-# This should work according to docs, but see RANGER-1565 -
-#
-# --RUN sed -i 's|#JAVA_HOME=|JAVA_HOME=/usr/lib/jvm/default-java|' install.properties
-# --RUN ./setup.sh
-#
-# So this is the workaround
-RUN JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 ./setup.sh
-
-# setup the ranger server
-# Note that we assume mariadb is running locally for policy storage - this will be the case in the kubernetes pod environment we create in the ranger helm charts
-# In future we may wish to get more configurable to better allow a standalone image to work and/or to split out the db into a seperate pod for workload management
-WORKDIR /opt/ranger-${build}-admin
-ENV		 dadir			 /opt/ranger-${build}-admin
-
-RUN sed -i "s|^db_root_password=|db_root_password=${rootpw}|g" "${dadir}/install.properties" 
-RUN sed -i "s|^db_password=|db_password=${rootpw}|g" "${dadir}/install.properties" 
-RUN sed -i "s|^db_host=localhost|db_host=localhost:3306|g" "${dadir}/install.properties" 
+WORKDIR /opt/ranger-admin
+RUN sed -i "s|^DB_FLAVOR=MYSQL|DB_FLAVOR=POSTGRES|g" install.properties
+RUN sed -i "s|^SQL_CONNECTOR_JAR=/usr/share/java/mysql-connector-java.jar|SQL_CONNECTOR_JAR=/usr/share/java/postgresql-jdbc.jar|g" install.properties
 RUN sed -i "s|^audit_solr_urls=|audit_solr_urls=http://localhost:6083/solr/ranger_audits|g" install.properties
-RUN sed -i "s|^rangerAdmin_password=|rangerAdmin_password=${rootpw}|g" "${dadir}/install.properties"
-RUN sed -i "s|^rangerTagsync_password=|rangerTagsync_password=${rootpw}|g" "${dadir}/install.properties"
-RUN sed -i "s|^rangerUsersync_password=|rangerUsersync_password=${rootpw}|g" "${dadir}/install.properties"
-RUN sed -i "s|^keyadmin_password=|keyadmin_password=${rootpw}|g" "${dadir}/install.properties"
 
-# Expose web ports
-EXPOSE          6080 6182 6083 6183
+# Create minimal runtime image
```

Comment:
One example here ... we could probably leave this issue open -- I think we could probably look through how a consumer may use the metadata. One specific example I was thinking of was the build date (even though we have the git history on disk)

Reply:
Per #1048 the build date is already tracked by Docker itself (as are the exposed ports)...  I'm not keen on tracking the same metadata in duplicate ways that then either must be kept in sync or risk confusing consumers as to which of the conflicting metadata is correct (?)

Reply:
will close as review and consider discussion in #1048